src/mn_setup.c

```diff
@@ -4696,7 +4696,7 @@ int MN_StringWidth(const char *string)
             continue;
         }
         c = M_ToUpper(c) - HU_FONTSTART;
-        if (c < 0 || c > HU_FONTSIZE || hu_font[c] == NULL)
+        if (c < 0 || c >= HU_FONTSIZE || hu_font[c] == NULL)
```

Comment:
This will still be wrong.

Reply:
Hmm, why?

Reply:
Sorry, I read it the other way round. I thought we had already changed this check to `c >= HU_FONTSIZE` everywhere.